__tests__/api.ts

```diff
@@ -1,42 +1,55 @@
 import { api, fetchApi } from '../src/api'
-import { mockFetch, FetchMock } from './_helper'
+import { FetchMock, serverMock, returnResponseText, returnResponseJson } from './_helper'
 
 describe('api()', () => {
-  let fetchApi: FetchMock
+  test('uses fetch() for requests to the serlo api', async () => {
+    serverMock(
+      'https://api.serlo.org/graphql',
+      returnResponseText('<api-result>')
+    )
```

Comment:
This should be moved into a `beforeEach()` function as it was the case before...